CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+### Setting dev env:
```

Comment:
I'd rather not have this file. also to run tests: `npm test`. this is true for every node module. also I don't want to promote installing mocha globally.